api-tester/src/main/java/com/revenuecat/apitester/java/PurchasesCommonAPI.java

```diff
@@ -125,6 +125,15 @@ static void checkConfiguration(final Context context,
         Purchases.setProxyURL(new URL(""));
         final URL proxyURL = Purchases.getProxyURL();
 
+        PurchasesConfiguration build = new PurchasesConfiguration.Builder(context, "")
+                .appUserID("")
+                .observerMode(true)
+                .observerMode(false)
+                .service(executorService)
+                .diagnosticsEnabled(true)
+                .entitlementVerificationMode(EntitlementVerificationMode.INFORMATIONAL)
+                .build();
```

Comment:
This isn't really needed for the `customEntitlementComputation` flavor, but moving api tests to the `main` source set since now this class will be common for both.